config/routes.rb

```diff
@@ -10,6 +10,9 @@
       devise_scope :user do
         get :status, to: 'api#status'
       end
+      namespace :user do
+        resources :matches, only: :index
```

Comment:
Is there any specific reason to put this in a separate namespace? I think you could move this inside the devise_scope from above.

Reply:
If I move `resources :matches, only: :index` inside devise_scope, then Rails generates this route `api/v1/matches#index` instead of `api/v1/user/matches#index`. Is there any other way to resolve this?